_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Three of these five buttons can be shown in the compact notification (remember that little arrow you use to expand/contract the notification?).
 
-For users on **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As we've noted above, MediaStyle notifications adapt to the colours of the cover art, which looks just plain awesome.
+For the betrodden of **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As noted above, MediaStyle notifications adapt to cover art colours. Big.
 
-For users on **Android 10+**: Seekbar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and have fun seeking!
+For the modern human using **Android 10+**: SeekBar! The old notification had a progress bar. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and you shall find what you seek.
 
-For users on **Android 11**: Congratulations! You get a notification now! NewPipe's media session will show up in the notification shade and in Quick Settings just as advertised.
+For the beyond human users of **Android 11**: Congratulations! You get a notification! NewPipe's media session shows up in the notification shade and in "Quick Settings", just as advertised. Enjoy the envy of the world.
 
 <div class="media-row media-items-2">
 <img class="no-flow img-responsive" style="align-self: center" src="/img/screenshots/notification_9.png"/>
 <img class="no-flow vertical" src="/img/screenshots/notification_11.png"/>
 </div>
-This cool contribution was brought to us in its original form by [@cool-student](https://github.com/cool-student). Sadly, we couldn't get in touch with them when we were ready to focus on this PR, so [@wb9688](https://github.com/wb9688) and [@Stypox](https://github.com/Stypox) swooped in to save the day! [@wb9688](https://github.com/wb9688) updated the code to conform to our standards and coding style, and [@Stypox](https://github.com/Stypox) did the (painstaking!) work of rebasing the code after the unified player PR was merged into the development branch.
+This cool contribution, brought to you in original form by [@cool-student](https://github.com/cool-student). When ready to focus on this PR, no means of contact could be established, so [@wb9688](https://github.com/wb9688) and [@Stypox](https://github.com/Stypox) swooped in to save the day! [@wb9688](https://github.com/wb9688) updated the code to conform to our standards and coding style, and [@Stypox](https://github.com/Stypox) did the (painstaking!) work of rebasing the code after the unified player PR was merged into the development branch.
 
 Ultimately, he took over the responsibility for this PR. He updated and polished the UI, fixed bugs, got everyone and their grandfather to test the changes, and pulled us to the finish line!
```

Comment:
Double "he". Polished the UI how? OK, nice, this is a story with an end, and a nice one. Good.